.github/workflows/README.md

```diff
@@ -271,7 +271,6 @@ Please note that jobs with matrix need to have matrix element in the comment. Ex
 
 | Workflow name | Matrix | Trigger Phrase | Cron Status |
 |:-------------:|:------:|:--------------:|:-----------:|
-| [ PostCommit BeamMetrics Publish ](https://github.com/apache/beam/actions/workflows/beam_PostCommit_BeamMetrics_Publish.yml) | N/A |`Run Beam Metrics Deployment`| [![.github/workflows/beam_PostCommit_BeamMetrics_Publish.yml](https://github.com/apache/beam/actions/workflows/beam_PostCommit_BeamMetrics_Publish.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/beam_PostCommit_BeamMetrics_Publish.yml?query=event%3Aschedule)
```

Comment:
BeamMetrics Publish and Website Publish are not really "postcommit" or test suites. It **deploys** the Beam website or **deploys** the community metrics dashboard. They should not have comment trigger (which could be used to attack beam website). So renamed them and move to "other" category.